packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -846,6 +853,27 @@ class EnabledExample extends React.Component<{}> {
   }
 }
 
+class TextInputExamples extends React.Component<$FlowFixMeProps, any> {
+  constructor(props) {
+    super(props);
+    this.state = {focused: false};
+  }
+  render() {
+    return (
+      <View>
+        <TextInput
+          accessible={true}
+          accessibilityState={{selected: this.state.focused}}
```

Comment:
cc @blavalla - Should the TextInput component always announce "selected" if it's focused and always announce "not selected" if it's blurred? Should this behavior be built into the component?

Reply:
I think there is some confusion here as to the usage of the "selected" property, which is not surprising given its vague name, and how other things (like text within the input for example) can also be "selected"!

This property is meant to represent an element that is highlighted visually for some reason. Maybe there's a list of text inputs, and only one of them is active at a time for some reason, and is highlighted in yellow to show this visually.  This item would be "selected", while the others would be "not selected".  

It doesn't really pertain to focus being within the input itself, or whether that input is currently being edited, or anything like that.

This same selected property can be applied to any type of view, and the issue here is simply that it doesn't work for `<TextInput>` like it does for `<View>`.

Reply:
Oh I see, thank you for the explanation! So the goal of the issue is just to *allow* the selected state to be set at the user's discretion, and not to always set it at any one time.